client/components/app.jsx

```diff
@@ -45,23 +55,29 @@ class App extends React.Component {
   }
   render () {
     return (
-      <div id="appBox">
-        <div id="reserveBox">
-          <h3 id="title">Make a reservation</h3>
-          <div id="partySize">
-            <PartySize partyChange={this.partySizeChange.bind(this)} />
-          </div>
+      <div id="component" className="col-lg-5 col-md-5 col-xs-10">
+        <div className="row">
+          <h4 id="title">Make a reservation</h4>
```

Comment:
Maybe make these ids (title/component) more specific? They seem to be pretty generic and in a larger project where there will be multiple titles / components it may be harder to keep track and not reuse.